metrics_utility/automation_controller_billing/collector.py

```diff
@@ -53,7 +54,12 @@ def gather(self, dest=None, subset=None, since=None, until=None, billing_provide
         if not self.is_enabled():
             return None
 
-        with self._pg_advisory_lock('gather_automation_controller_billing_lock', wait=False) as acquired:
+        key = 'gather_automation_controller_billing_lock'
+        suffix = os.environ.get('METRICS_UTILITY_COLLECTOR_LOCK_SUFFIX')
+        if suffix:
+            key = f'gather_automation_controller_billing_{suffix}_lock'
+
+        with self._pg_advisory_lock(key, wait=False) as acquired:
```

Comment:
How this is intented to be used? Each process will define its own lock? What is the purpose of lock then, maybe the logic should be to avoid locking here completely?

Reply:
Ladas told me that I need a separate lock to avoid a cronjob blocking the other cronjob. The job_host_summary will continue. to run in the hourly cronjob generated by the operator. The CPU collection will run on a new cronjob every 5 min. The new cronjob will not run the job_host_summary.
Is this clarifying your question?

Reply:
That confirms it, thanks :-)